### ISSUE TITLE
ADA Accessibility: NVDA does not read members button correctly in CHR

### ISSUE THREAD

**Issue by jbattistispiria** [mattermost/mattermost-server#20026]
#### Summary
The members button in the channel header does is not read correctly with screen reader.
![image](https://user-images.githubusercontent.com/103535117/164010363-c1aa7824-179a-4d1d-a87c-037ba019e084.png)

#### Steps to reproduce
1. Run a screen reader program like NVDA.
2. Press CTRL+F6 (Windows) to navigate to the channel header region
3. Press TAB to navigate through the options till members button.

#### Expected behavior
On members button, NVDA say: "Members".

#### Observed behavior (that appears unintentional) 
On members button, NVDA say: "2 button, Members".

#### Possible fixes
There's a missing aria-label on the members button.

**Comment by jbattistispiria** [comment]
I can take it, i found it while working on another accessibility issue.
